cadc-datalink/src/test/resources/valid-template.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I forgot to comment on the valid example. I'd prefer to use generic values rather than our current service values.

Change the value of the INFO element to `placeholder: will be removed`

Only one of resourceIdentifier and accessURL is needed, so maybe 
- remove the resourceIdentifier
- change the accessURL to `https://example.net/datalink/links` (change arraysize to `*`)

Keep the standardID as-is.

Let's leave the ID and ref as-is as well; promote CAOM a bit :-)

Simplify and make other tets files generic as well.